javascript/pkce_oauth/server/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "asana-deprecation-alerts",
+  "version": "0.0.1",
+  "description": "Automated Asana Task creation anytime a new deprecation period has started.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [
+    "asana",
+    "deprecation"
+  ],
+  "author": "Ross Grambo",
+  "license": "ISC",
+  "dependencies": {
+    "asana": "^0.17.1",
+    "cookie-parser": "^1.4.4",
+    "cors": "^2.8.5",
+    "express": "^4.17.1",
+    "firebase-admin": "^8.3.0",
+    "firebase-functions": "^3.2.0",
+    "js-sha256": "^0.9.0",
```

Comment:
[Minor] You don't need a dependency for this if you don't want it. See [these docs](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/digest).